components/contextualhelp/stories/contextualhelp.test.js

```diff
@@ -22,5 +22,13 @@ export const ContextualHelpGroup = Variants({
 				url: "#",
 			},
 		},
+		{
+			testHeading: "Help",
+			iconName: "Help",
+			customStyles: {
+				"inline-size": "275px",
+				"margin-bottom": "170px",
+			},
+		},
```

Comment:
I ran VRT against `main` originally to capture this test, but haven't run it against `s2-foundations-redux` yet.